t/Makefile.am

```diff
@@ -1,47 +1,9 @@
-UPDIR = $(dir $(PWD))
```

Comment:
is the complete file even required now? I guess we can delete it

Reply:
ok, I deleted it

Reply:
Now the build is faling with:
```
configure.ac:52: error: required file 't/Makefile.in' not found
Makefile.am: installing './depcomp'
autoreconf: automake failed with exit status: 1
```
https://travis-ci.org/os-autoinst/os-autoinst/builds/622425938#L525

Reply:
added it back again, empty